cmd/releasego/update-azure-linux.go

```diff
@@ -96,19 +116,88 @@ func updateAzureLinux(p subcmd.ParseFunc) error {
 		return err
 	}
 
-	ar := txtar.Archive{
-		Comment: []byte("Bump version to " + assets.GoVersion().Full()),
-		Files: []txtar.File{
-			{Name: cgManifestFilepath, Data: cgManifestBytes},
-			{Name: golangSignaturesFilepath, Data: golangSignaturesFileBytes},
-			{Name: golangSpecFilepath, Data: []byte(golangSpecFileContent)},
-		},
+	ref, _, err := client.Git.GetRef(ctx, owner, repo, baseBranch)
+	if err != nil {
+		return fmt.Errorf("failed to get ref: %v", err)
+	}
+
+	newRef := &github.Reference{
+		Ref:    github.String(updateBranch),
+		Object: &github.GitObject{SHA: ref.Object.SHA},
+	}
+
+	if _, _, err = client.Git.CreateRef(ctx, owner, repo, newRef); err != nil {
+		return fmt.Errorf("Failed to create ref: %v", err)
+	}
+
+	updatedFiles := map[string][]byte{
+		golangSpecFilepath:       []byte(golangSpecFileContent),
+		golangSignaturesFilepath: golangSignaturesFileBytes,
+		cgManifestFilepath:       cgManifestBytes,
+	}
+
+	for filePath, newContent := range updatedFiles {
+		// Get the file to update
+		file, _, _, err := client.Repositories.GetContents(ctx, owner, repo, filePath, &github.RepositoryContentGetOptions{Ref: updateBranch})
+		if err != nil {
+			return fmt.Errorf("Failed to get file %q: %v", filePath, err)
+		}
+
+		// Update the file
+		_, _, err = client.Repositories.UpdateFile(ctx, owner, repo, filePath, &github.RepositoryContentFileOptions{
+			Message: github.String(generatePRTitleFromAssets(assets)),
+			Content: newContent,
+			SHA:     file.SHA,
+			Branch:  github.String(updateBranch),
+		})
+		if err != nil {
+			return fmt.Errorf("Failed to update file %q: %v", filePath, err)
```

Comment:
```suggestion
			return fmt.Errorf("failed to update file %q: %v", filePath, err)
```

A few more capital errors earlier in the file, too.